### DISCUSSION TITLE
App version in Helm chart is not updated with new releases

### DISCUSSION THREAD

**Discussion by elchenberg** [aquasecurity/trivy#4683] [Ideas]
### Description

When a new version of Trivy gets released the appVersion field in the Chart.yaml of the Helm chart is not updated accordingly. The latest release is v0.42.1 but the Helm chart still uses v0.37.2. In the past I have open pull requests to update the field manually but I think this task should be integrated into the release process somehow.

### Desired Behavior

The appVersion in the chart's Chart.yaml gets updated every time there is a new Trivy release.

### Actual Behavior

The chart uses an old version of Trivy.

### Reproduction Steps

```bash
1.
2.
3.
...
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A
```

### Operating System

N/A

### Version

```bash
N/A
```

### Checklist

- [ ] Run `trivy --reset`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by AnaisUrlichs** [comment]
@chen-keinan how could we automate this -- maybe run an Action on the Trivy Operator side that makes the comparison?

**Reply by chen-keinan** [reply]
I think @elchenberg  refering to trivy server [chart app version](https://github.com/aquasecurity/trivy/blob/main/helm/trivy/Chart.yaml#L4)
and I assume it can automated it as part of release process/ or else , there was an issue with helm test which failed as it wasn't running on the latest tag

**Reply by AnaisUrlichs** [reply]
ahhh
